.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,6 @@
 <!--
-Hi there! thank you for discovering and submitting an issue!
+Hi there!
+iothub-explorer will be retired on November 31st, 2018 and has been replaced by the [Azure CLI IoT Extension](https://aka.ms/iotcli).
 
-Please first tell us a little bit about the environment you're running:
-The commands in the comments can be run directly in a command prompt.
+We will not address any issue regarding iothub-explorer except those related to the migration to the azure cli.
```

Comment:
consider replacing text with 'We will not address any issues regarding iothub-explorer except for those related to migrating towards the Azure CLI.'